pkg/process/checks/templates/processes.tmpl

```diff
@@ -0,0 +1,35 @@
+Processes
+=========
+{{- range $process := .Processes}}
+> PID: {{ .Pid }} NSPID: {{ .NsPid }} PPID: {{ .Command.Ppid }}
+  Container ID: {{ .ContainerId }}
+  Exe: {{ .Command.Exe }}
+  Args:{{ range .Command.Args}} '{{.}}'{{ end }}
+  Cwd: {{ .Command.Cwd }}
+{{- with .User }}
+  User: {{.Name}} Uid: {{.Uid}} Gid: {{.Gid}} Euid: {{.Euid}} Egid: {{.Egid}} Suid: {{.Suid}} Sgid: {{.Sgid}}
+{{- end }}
+  Create Time: {{ .CreateTime }}
```

Comment:
We're missing the `State` field here
https://github.com/DataDog/datadog-agent/pull/11487/commits/55736e04d7ddb5bb47bbb61d9914f659941c11e7

Reply:
Added thank you!